handler/core/src/main/java/io/knotx/fragments/task/options/ActionNodeConfigOptions.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.fragments.task.options;
+
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.json.JsonObject;
+import java.util.Objects;
+
+/**
+ * Action node configuration model. It is model for {@link NodeOptions#getConfig()} JSON object.
+ *
+ * <pre>
+ * node {
+ *   factory = action
+ *   config {
+ *     HERE
```

Comment:
??

Reply:
HERE - means that this JSON node is used in ActionNodeConfigOptions(json).

Reply:
So maybe...:

```hocon
config { //represented by ActionNodeConfigOptions
   ...
}
```
?